src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -228,14 +212,15 @@
             <v-col
               cols="12"
               sm="9"
+              class="file-buttons pt-2 pt-sm-0"
             >
               <!-- the director's affidavit file -->
               <template v-if="isContinuationInAffidavitRequired">
                 <v-btn
                   v-if="getExistingBusinessInfo.affidavitFileName"
                   text
                   color="primary"
-                  class="download-affidavit-btn mt-sm-n2 d-block"
+                  class="download-affidavit-btn mt-sm-n2 d-block px-2 ml-n2"
```

Comment:
This and above : fixed alignment, ie

![image](https://github.com/user-attachments/assets/ec4aef5e-132c-4cd2-8c80-9b900a6c1613)